tests/publiccloud/storage_perf.pm

```diff
@@ -72,11 +72,12 @@ sub run {
 
     my $provider = $self->provider_factory();
     my $instance = $provider->create_instance(use_extra_disk => {size => $disk_size, type => $disk_type});
+    $instance->wait_for_guestregister() if get_var('FLAVOR') !~ m/BYOS/;
 
     $tags->{os_kernel_release} = $instance->run_ssh_command(cmd => 'uname -r');
     $tags->{os_kernel_version} = $instance->run_ssh_command(cmd => 'uname -v');
 
-    $instance->run_ssh_command(cmd => 'sudo SUSEConnect -r ' . $reg_code,                timeout => 600) if $reg_code;
+    $instance->run_ssh_command(cmd => 'sudo SUSEConnect -r ' . $reg_code,                timeout => 600) if (get_required_var('FLAVOR') =~ m/BYOS/);
```

Comment:
I think this will fail in early stage of the image release process. As we do not have scc-proxy.
I currently don't have a solution to this problem.

Reply:
but we do the same un LTP, right? So it will fail there as well... Let's just leave until we have a good solution...